### PR TITLE
update

### PR DESCRIPTION
**Pull request by KokuoU1210**
update readme

### CODE REVIEW

**Comment by KokuoU1210** [comment]
style: update readme

File: README.md [hunk 1]
```diff
@@ -36,4 +36,48 @@ A first guide on how to use (i.e. adapt, measure, optimize) CrowdNav with the [R
...
+      def update(self, P, I, D):
+          self.P = P
+          self.I = I
+          self.D = D
+          
+### Simulation
+
+* PID controller is used in `Simulation.py`. The simulation start from time tick 0 and keeps coolecting values of each parameters, in every loop we decide whether to add/remove cars and how many cars to add/remove. If the actual arrived cars are less than the PID controller's expectation, we add cars, otherwise we remove cars. This simulation reports current status to Kafka every 30 ticks and print status update every 100 ticks if current running mode is not parallel.
+
+      if (cls.tick % 100) == 0 and Config.parallelMode is False:
```
Comment (fif911) [inline README.md]: Remove this if. It's just prints to conlose

File: app/simulation/Simulation_Seasonality.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
+import json
```
Comment (fif911) [inline app/simulation/Simulation_Seasonality.py]: Remove this file :)

**Review by fif911 — CHANGES_REQUESTED** [review]

**Comment by KokuoU1210** [comment]
style: update readme

**Review by fif911 — APPROVED** [review]

**Comment by fif911** [comment]
Great 👍

**Pull request merged**